docs/Installing.asciidoc

```diff
@@ -411,10 +411,12 @@ connecting to a remote PostgreSQL database with a username and password:
 [source, ini]
 --------------------------------------------------------------------------------
 [production]
-dsn = dbi:Pg:dbname=openqa;host=db.example.org
+dsn = dbi:Pg:dbname=openqa
```

Comment:
Right. but it's very unfortunate that this is the only example when most users will not have an extra DB server handy. So they will try host=localhost and this will fail - as postgresql defaults to local auth.